doc/legacy_mode/linting.md

```diff
@@ -0,0 +1,27 @@
+# Legacy Mode Linting Page

```

Comment:
I think adding "Page" to the end of each documentation page is unnecessary. `Linting in Legacy Mode` I believe describes the page better.